certbot_dns_desec/dns_desec.py

```diff
@@ -86,8 +87,27 @@ def __init__(self, endpoint, token):
         self.session.headers["Authorization"] = f"Token {token}"
         self.session.headers["Content-Type"] = "application/json"
 
+    @staticmethod
+    def desec_request(method, **kwargs):
+        response: requests.Response = method(**kwargs)
+        if response.status_code == 429 and 'Retry-After' in response.headers:
+            try:
+                cooldown = int(response.headers['Retry-After'])
```

Comment:
For the record, I tested by running something every .3 seconds that `Retry-After: 0` does not occur (min value is `1`). (I was afraid that 0.4s would be rounded to 0, so that we'd have to add half a second here.)